packages/core/src/bind/index.ts

```diff
@@ -23,7 +24,7 @@ import { EMPTY_VALUE } from "../internal/empty-value"
 export function bind<T>(
   observable: Observable<T>,
   defaultValue?: T,
-): [() => Exclude<T, typeof SUSPENSE>, Observable<T>]
+): [() => Exclude<T, typeof SUSPENSE>, StateObservable<T>]
```

Comment:
Should we expose the `DeafultedStateObservable` with the defaultValue overload on this release?

Reply:
I pushed a commit adding them in

Reply:
Awesome!